### PR TITLE
update kubelet args for max-pods as part of the upgrade

### PR DESCRIPTION
**Pull request by ibrokethecloud**
**IMPORTANT: Please do not create a Pull Request without creating an issue first.**

This PR is for the upgrade path for issue https://github.com/harvester/harvester/issues/2707

**Problem:**

**Solution:**

**Related Issue:**

**Test plan:**

### CODE REVIEW

File: package/upgrade/upgrade_node.sh [hunk 1]
```diff
@@ -605,6 +615,8 @@ command_single_node_upgrade() {
 
   convert_nodenetwork_to_vlanconfig
 
+  # update max-pods to 200 #
+  set_max_pods
```
Comment (bk201) [inline package/upgrade/upgrade_node.sh]: Should we move this before `upgrade_rke2` function?
Putting the call here then the change takes effect after rebooting. In theory there is no difference.

Comment (ibrokethecloud) [reply package/upgrade/upgrade_node.sh]: sounds good. makes it better to organize the rke2 changes together.

**Review by Vicente-Cheng — APPROVED** [review]
LGTM, thanks!

**Review by bk201 — APPROVED** [review]
LGTM!

**Pull request merged**